src/CSSTransition.js

```diff
@@ -198,6 +201,10 @@ class CSSTransition extends React.Component {
     if (doneClassName) {
       removeClass(node, doneClassName);
     }
+    // Remove class atribute if it is empty
```

Comment:
do we need this change? this seems like essentially a separate thing. does it hurt to leave an empty `class` attribute around? we wouldn't know if e.g. the node originally had a `class=""`, would we?

Reply:
@taion youp. I don't know if it is right approach. I add it to cleanup for this specific case, cause it just add class to div, like so:
```html
<div class></div>
<div class></div>
<div class></div>
``` 
Maybe there is another way to fix this?
It doesn't hurt to add `class` attribute, so I can delete this part

Reply:
I think it's harmless.